src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -169,31 +170,11 @@ public static FlowExecutionList get() {
      */
     @Extension
     public static class ItemListenerImpl extends ItemListener {
-        @Inject
```

Comment:
check for unused imports

Reply:
I removed another unnecessary usage of `@Inject` along with the import in ec57f3d.